draft-ietf-quic-transport.md

```diff
@@ -3666,17 +3644,33 @@ validation at a sender and including a lower value than what was included in a
 previous ACK frame could cause ECN to be unnecessarily disabled; see
 {{ecn-validation}}.
 
+{{ack-tracking}} describes an exemplary approach for determining what packets
+to acknowledge in each ACK frame.  Though the goal of this algorithm is to
+generate an acknowledgment for every packet that is processed, it is still
+possible for acknowledgments to be lost.  A sender cannot expect to receive
```

Comment:
Minor nit; I think the sender is right to expect this in the normal case, but it's not guaranteed and can't be relied upon.

Reply:
Thanks, I moved this up and tweaked the wording, since it's a general statement and not specific to ack-tracking.